src/import-data/useImportRequest.ts

```diff
@@ -140,10 +134,6 @@ const getTDRSnapshotReferenceImportRequest = async (
     throw new Error('Unable to load snapshot.');
   }
 
-  if (snapshot.cloudPlatform === 'azure') {
-    throw new Error('Importing by reference is not supported for Azure snapshots.');
```

Comment:
I'm wondering if we should continue to throw this error?  Otherwise if someone for some reason tries to import an azure snapshot it'll make the attempt and presumably error elsewhere

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5275/commits/af6b84e09e01c2b7377055df0075b48de5847440